.circleci/config.yml

```diff
@@ -131,7 +141,7 @@ workflows:
       - sonar-scan:
           context: SonarCloud
           requires:
-            - install-dependencies
+            - tests
```

Comment:
Requires test rather than in parallel because it relies on the coverage reports.